app/screens/ServerView.tsx

```diff
@@ -194,6 +194,8 @@ const ServerView: React.FC<{
           selectionColor={colors.text.secondary}
           textContentType="URL"
           placeholder="http://demo.navidrome.org"
+          autoCorrect="false"
+          keyboardType="url"
```

Comment:
The `keyboardType` of `url` is an iOS only thing so it won't have any effect here and should be removed: https://reactnative.dev/docs/0.64/textinput#keyboardtype

Reply:
Do you mean you don’t build for iOS? Because it would at least do something there?

Reply:
I don't, subtracks is an android only app for now as I don't have any Apple devices here to work on/maintain an iOS release.

Reply:
Right. Neither do I, actually 🙂